runtime/gc_glue_java/HeadlessMixedObjectScanner.hpp

```diff
@@ -31,11 +31,11 @@ class GC_HeadlessMixedObjectScanner : public GC_ObjectScanner
 {
 	/* Data Members */
 private:
+	fomrobject_t *_slotPtr; /**< pointer to first slot in current scan segment */
```

Comment:
Perhaps you can be even more descriptive: _slotObjectPtr (to clearly contrast from _slotMapPtr). 

Going further, _slotObjectPtr could be called _startSlotObjectPtr

If so, then  _endPtr would be _endSlotObjectPtr